website/landing/components/common/Clipboard.tsx

```diff
@@ -53,14 +43,16 @@ export const Clipboard: React.FC = () => {
   const [toastVisible, setToastVisible] = useState(false);
 
   const copyToClipboard = useCallback(() => {
-    navigator.clipboard.writeText(command);
-    setToastVisible(true);
+    try {
+      navigator.clipboard.writeText(command);
+      setToastVisible(true);
+    } catch {}
```

Comment:
I think Eslint is complaining about this

Reply:
@danilowoz thanks for the catch! just fixed it here: fd43249